modules/ROOT/pages/view-partner-details.adoc

```diff
@@ -0,0 +1,14 @@
+= View a Partner's Details
+
+After you create a partner, you can add, update, or delete partner details:
+
+. In Partner Manager, select *Partners* in the navigation menu to access the *Partners* page.
+. Click the partner to view its details and make the following updates.
```

Comment:
Sometimes we refer to "the partner" as "they/them," and here "it". Are we making a distinction between the collective personnel and the software item?
(I would refer to a collection of people as an "it": a club, a business, and so on. But others disagree.)